tensilelite/Tensile/Components/MAC_F64.py

```diff
@@ -45,11 +48,18 @@ def __call__(self, writer, m, innerUnroll):
                 vars["a"] = a
                 for iui in range(0, innerUnroll):
                     vars["iui"] = iui
-                    cStr        = "v[vgprValuC+({a}+{b}*{ThreadTile0})*2:(vgprValuC+{a}+{b}*{ThreadTile0})*2+1]".format_map(vars)
-                    aStr        = "v[vgprValuA_X{m}_I{iui}+{a}*2:vgprValuA_X{m}_I{iui}+{a}*2+1]".format_map(vars)
-                    bStr        = "v[vgprValuB_X{m}_I{iui}+{b}*2:vgprValuB_X{m}_I{iui}+{b}*2+1]".format_map(vars)
-                    module.addInst("v_fma_f64", cStr, aStr, bStr, cStr, "")
-                    module.add(priority(writer, 1, "Raise priority while processing macs"))
 
-        module.add(priority(writer, 0, "Reset priority after macs"))
+                    # cStr = ValuC+({a}+{b}*{ThreadTile0})*2
+                    # aStr = ValuA_X{m}_I{iui}+{a}*2
+                    # bStr = ValuB_X{m}_I{iui}+{b}*2
+                    cStr = "ValuC+%d" % ((vars["a"]+vars["b"]*vars["ThreadTile0"])*2)
+                    aStr = "ValuA_X%d_I%d+%d" % (vars["m"], vars["iui"], vars["a"]*2)
+                    bStr = "ValuB_X%d_I%d+%d" % (vars["m"], vars["iui"], vars["b"]*2)
+
+                    module.add(VFmaF64(dst=vgpr(cStr, 2), src0=vgpr(aStr, 2),
+                                       src1=vgpr(bStr, 2), src2=vgpr(cStr, 2)))
+                    module.add(SSetPrior(prior=1, comment="Raise priority while processing macs"))
```

Comment:
should this only raise one time after the first FmaF64 valu?
FmaF64 //first FmaF64
Set_priority 1
FmaF64
....
FmaF64
FmaF64 //last FmaF64
Set_priority 1